support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/support/oauth/web/response/callback/OAuth20TokenAuthorizationResponseBuilder.java

```diff
@@ -57,10 +57,17 @@ public ModelAndView build(final JEEContext context,
         val accessToken = result.getAccessToken().orElse(null);
         val refreshToken = result.getRefreshToken().orElse(null);
         LOGGER.debug("Generated OAuth access token: [{}]", accessToken);
+        OAuth20JwtAccessTokenEncoder.builder()
```

Comment:
The move of OAuth20AccessTokenEncoder.builder() is to fix a null point exception in OidcImplicitIdTokenAuthorizationResponseBuilder (subclass of OAuth20TokenAuthorizationResponseBuilder) caused by the commit 76005b942b2f84689fa498cf8af8572e8d18a13c. This commit expects OAuth20JwtAccessTokenEncoder.encode() to be called before OidcIdTokenGeneratorService.generate().